elk/utils_evaluation/utils_evaluation.py

```diff
@@ -58,14 +61,47 @@ def organize(hidden_states, mode):
     raise NotImplementedError("This mode is not supported.")
 
 
-def normalize(data, scale=True, demean=True):
-    # demean the array and rescale each data point
-    data = data - np.mean(data, axis=0) if demean else data
-    if not scale:
-        return data
-    norm = np.linalg.norm(data, axis=1)
-    avgnorm = np.mean(norm)
-    return data / avgnorm * np.sqrt(data.shape[1])
+def normalize(
+    hidden_states,
+    permutation,
+    scale: Literal["original", "none", "elementwise", "projection"] = "original",
+    demean=True,
```

Comment:
Add a boolean include_test_set flag to use the original normalization without splitting test and train set